workos/types/directory_sync/directory_user.py

```diff
@@ -0,0 +1,38 @@
+from typing import List, Literal, Optional
+
+from workos.resources.workos_model import WorkOSModel
+
+
+DirectoryUserState = Literal["active", "inactive"]
+
+
+class DirectoryUserEmail(WorkOSModel):
+    type: Optional[str] = None
+    value: Optional[str] = None
+    primary: Optional[bool] = None
+
+
+class Role(WorkOSModel):
+    slug: str
```

Comment:
We could reuse this for org. memberships, so I may end up moving it.

Reply:
Yup, I figured it might move around